input/fsh/Bundle.fsh

```diff
@@ -43,6 +80,26 @@ Description: "A collection of data for an mCODE cancer patient."
 * entry[cancerGenomicsReport].resource only CancerGenomicsReport
 * entry[geneticSpecimen].resource only GeneticSpecimen
 * entry[genomicRegionStudied].resource only GenomicRegionStudied
+* entry[cancerRelatedComorbidities].resource only CancerRelatedComorbidities
+* entry[vitalSign].resource only VitalSign
+* entry[coreLaboratory].resource only CoreLaboratory
+* entry[otherResources].resource only OtherResources
+
+
+Profile: CoreLaboratory
+Parent: USCoreObservationLab
+Id: mcode-core-laboratory
+Description: "Abstract profile for discrimination of the core laboratory slice in the mCODE Patient Bundle."
+* ^abstract = true
+* code from CoreLaboratoryVS (required)
+
+Profile: OtherResources
+Parent: Bundle
+Id: mcode-other-resources
+Description: "Abstract profile for to define the 'other resources' bundle in the mcode patient bundle."
+* ^abstract = true
```

Comment:
Do we need an example for this? (Not necessarily for this PR but in general.)

Reply:
I'm adding one now.

Reply:
Pushing...